libraries/aws_ec2_instances.rb

```diff
@@ -35,16 +38,18 @@ def fetch_data
         @api_response = @aws.compute_client.describe_instances(pagination_options)
       end
       return [] if !@api_response || @api_response.empty?
-
       @api_response.reservations.each do |res|
-        instance_rows += res.instances.map do |instance|
-          {
+        res.instances.each do |instance|
+          tags = map_tags(instance.tags)
+          instance_rows += [{
             instance_id: instance.instance_id,
             vpc_id: instance.vpc_id,
             subnet_id: instance.subnet_id,
             instance_type: instance.instance_type,
-            tags: map_tags(instance.tags),
-          }
+            tags: tags,
```

Comment:
Haven't dropped in here with the debugger but wouldn't this line potentially resolve to e.g.
```
map_tags(instance.tags): map_tags(instance.tags), 
```
Either way, might be preferable to use a distinct name for the instance tags.